BLSQ/iaso
# IA-2998  IA-2999 IA-3000 IA-3001 registry fixes

## IA-2998:
Eye symbole is at the same time :
- Details
- See registry

## IA-2999:
On staging (instance kqrxutg) I zoom from national to AS ==> No problem
When clicking on a upper level (e.g. burkina Faso) , the map remain with the old shape (although the legend is updated)

## IA-3000:
IASO should have one “refercence form” that will be the one that will be displayed first in the registry (withouth having to select it). Then other forms could also be selected throught the picker as it is already the case. 
If no reference submission but having another submission, preselect it

## IA-3001:
On staging, instance kqrxutg
I select the option “draw a square to zoom” (but not using it directly)
I open the map in big (or if it was in big, I reduce my map to small)
I’m stuck with my mouse cursor ready for drawing a square but can not draw it
==> I need to refresh my page to exit this

## Self proofreading checklist


## Changes

- allow to change link to org unit icon
-  extend custom zoom bar to be able to trigger `fitToBounds` on a key
- improve navigation and fit to bound
- preselect an existing submission if no reference instance is set to the selected org unit
- allow to navigate with breadcrumb in fullscreen


## How to test

Open registry page on a root org unit having children with shapes and submissions.
- Check icons on list view
- drill down and up trough org unit, make sure map is adapting to the org unit and that the selected org unit is correct
- click on an org unit having submissions but no reference one, first submission should be selected
-  select the option “draw a square to zoom”, click on fullscreen mode, you should be stuck anymore

## Print screen / video

IA-2998:
<img width="824" alt="Screenshot 2024-05-24 at 11 48 08" src="https://github.com/BLSQ/iaso/assets/12494624/12ccaf3c-cc66-4c07-9d7a-d122f07d84da">

IA-2999:
https://github.com/BLSQ/iaso/assets/12494624/133ed244-6035-4bf0-8762-58cbd2312be9

IA-3000:
https://github.com/BLSQ/iaso/assets/12494624/3746b826-8c2e-4d07-874e-a96e83ecf31c

IA-3001:
https://github.com/BLSQ/iaso/assets/12494624/49197ba0-734b-42b2-b224-495057b420fc
